packages/webapp/src/components/Task/TaskReadOnly/index.jsx

```diff
@@ -233,26 +233,30 @@ export default function PureTaskReadOnly({
         </div>
       )}
 
-      <Semibold style={{ marginBottom: '12px' }}>{t('TASK.LOCATIONS')}</Semibold>
-      {isTaskType(taskType, 'TRANSPLANT_TASK') && (
-        <TransplantLocationLabel
-          locations={task.locations}
-          selectedLocationId={task.selectedLocationIds[0]}
-          pinCoordinate={task.pinCoordinates[0]}
-        />
-      )}
-      <LocationPicker
-        onSelectLocation={() => {
-          //  TODO: fix onSelectLocationRef in LocationPicker
-        }}
-        readOnlyPinCoordinates={task.pinCoordinates}
-        style={{ minHeight: '160px', marginBottom: '40px' }}
-        locations={task.locations}
-        selectedLocationIds={task.selectedLocationIds || []}
-        farmCenterCoordinate={user.grid_points}
-        maxZoomRef={maxZoomRef}
-        getMaxZoom={getMaxZoom}
-      />
+      {task?.locations?.length || task?.pinCoordinates?.length ? (
```

Comment:
The null checks for task don't seem necessary, as they aren't needed elsewhere in the code, but it works great!

Reply:
I was hit with this irrational fear of being the cause of a new type error during bug bash 😅 But you're absolutely right -- there are about 20 lines of code that would crash before even getting here. I will remove them, thank you! 🙏

Reply:
I looked properly and the whole Pure component is conditionally rendered based on task existing 😜